### ISSUE TITLE
Attack Protection does not replace keyword mapping

### ISSUE THREAD

**Issue by iamjoeker** [auth0/auth0-deploy-cli#477]
### Description

Attack protection configuration files in directory mode do not replace keyword mappings. I am working on a PR to fix the issue now and will submit today.

### Reproduction

1. Create attack protection config (e.g., `./attack-protection/breached-password-detection.json`)
2. Replace one of the values with a keyword replacement pattern (see below for example)
3. Add the corresponding keyword to the `AUTH0_KEYWORD_REPLACE_MAPPINGS` in `config.json`
4. Run `a0deploy deploy -c config.json -i .` and note the errors when attempting to deploy the attack protection settings.

#### Example

* `attack-protection/breached-password-detection.json`:
  ```json
  {
    "enabled": "@@ATTACK_PROTECTION_ENABLED@@",
    "shields": [
      "admin_notification",
      "block",
      "user_notification"
    ],
    "admin_notification_frequency": [
      "immediately",
      "monthly",
      "daily",
      "weekly"
    ],
    "method": "standard"
  }
  ```
* `config.json`
  ```json
  ....
  "AUTH0_KEYWORD_REPLACE_MAPPINGS": {
    "ATTACK_PROTECTION_ENABLED": false
  }
  ....
  ```
* Error when running deploy
  ```
  2022-04-06T14:43:27.082Z - info: Updated [attackProtection]: {"breached-password-protection":{"enabled":"##ATTACK_PROTECTION_ENABLED##"},"brute-force-protection":{"enabled":"##ATTACK_PROTECTION_ENABLED##"},"suspicious-ip-throttling":{"enabled":"##ATTACK_PROTECTION_ENABLED##"}}
  (node:97253) UnhandledPromiseRejectionWarning: Bad Request: Payload validation error: 'Expected type boolean but found type string' on property enabled (Whether or not breached password detection is active).
      at <redacted>/.config/yarn/global/node_modules/rest-facade/src/Client.js:402:25
      at Request.callback (<redacted>/.config/yarn/global/node_modules/superagent/lib/node/index.js:905:3)
      at <redacted>/.config/yarn/global/node_modules/superagent/lib/node/index.js:1126:20
      at IncomingMessage.<anonymous> (<redacted>/.config/yarn/global/node_modules/superagent/lib/node/parsers/json.js:22:7)
      at IncomingMessage.emit (events.js:412:35)
      at IncomingMessage.emit (domain.js:475:12)
      at endReadableNT (internal/streams/readable.js:1334:12)
      at processTicksAndRejections (internal/process/task_queues.js:82:21)
  (Use `node --trace-warnings ...` to show where the warning was created)
  (node:97253) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
  (node:97253) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

### Environment

* Library Version: 7.6.0
* Node version: lts/ferium (v14.18.2)
* OS: macOS 12.3.1

**Comment by willvedd** [comment]
Thanks for reporting this. Funny enough, we noticed this during our routine TS migration work. It was addressed with this line: https://github.com/auth0/auth0-deploy-cli/pull/475/files#diff-bceb291f26ea163907ba340065f651ec98fbf3b892b093ffae2e96839f88c10eR43.

You can expect to have a patched release for this sometime next week once the above PR is verified and merged.

**Comment by iamjoeker** [comment]
@willvedd Awesome. Thanks for the reply. I'll leave my PR open in case you want to incorporate the test I added for keyword replacement in the attack protection asset.

**Comment by willvedd** [comment]
@iamjoeker Thanks for pushing a fix for this. I've included in the v7.7.1 release I just cut.
